packages/core/src/indexing/postgres/__tests__/read-csv-fixture.util.ts

```diff
@@ -2,8 +2,10 @@ import { IndexStreamArgs } from '../../database-index-api.js'
 import fs from 'node:fs'
 import csv from 'csv-parser'
 import { StreamID } from '@ceramicnetwork/streamid'
+import { CID } from 'multiformats/cid'
 
 const MODEL_ID = 'kjzl6cwe1jw145m7jxh4jpa6iw1ps3jcjordpo81e0w04krcpz8knxvg5ygiabd'
+const FAKE_CID = CID.parse('bafybeig6xv5nwphfmvcnektpnojts33jqcuam7bmye2pb54adnrtccjlsu')
```

Comment:
This CID is used in other places, while having no explicit connection between different (yet same) constants.
I see two options to fix that:
1) Export this thing and reuse in the tests.
2) Add another optional parameter to `readCsvFixture` function, so that a test can decide what to pass there.

I would encourage you to go with (2).

Reply:
in that case, wouldn't we want to do the same thing for the `MODEL` as well? So extend `readCsvFixture` to accept both `MODEL` & `CID` but not make it optional but force the tests to pass the values and make it a pure util function as a result?